src/Nethermind/Nethermind.TxPool/BlobCatcher.cs

```diff
@@ -0,0 +1,39 @@
+﻿// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using System.Linq;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.TxPool.Collections;
+
+namespace Nethermind.TxPool;
+
+public class BlobCatcher(BlobTxDistinctSortedPool blobPool)
```

Comment:
I don't get the naming, what are we catching? Maybe `BlobFinder`?

Reply:
`BlobFinder` was my first thought, but then `BlobCatcher` came to my mind, and I liked it - it sounded cool and a bit badass. I chose it, sacrificing accuracy for style. But after giving it more thought, it’s not as great as in first impression. Agree, Finder is more precise. Changed.